WalletWasabi.Gui/Controls/WalletExplorer/HistoryTabView.xaml

```diff
@@ -41,7 +41,7 @@
                   </Border>
                   <TextBlock Classes="monospaceFont" Text="{Binding DateTime, ConverterParameter=11, Converter={StaticResource LurkingWifeModeStringConverter}, Mode=OneWay}" Grid.Column="1" />
                   <TextBlock Classes="monospaceFont" Text="{Binding AmountBtc, ConverterParameter=8, Converter={StaticResource LurkingWifeModeStringConverter}, Mode=OneWay}" Grid.Column="2" Foreground="{Binding AmountBtc, Converter={StaticResource MoneyBrushConverter}}" />
-                  <TextBlock Classes="monospaceFont" IsVisible="{Binding !ClipboardNotificationVisible}" Text="{Binding TransactionId, ConverterParameter=50, Converter={StaticResource LurkingWifeModeStringConverter}, Mode=OneWay}" Grid.Column="3" />
+                  <controls:ExtendedTextBox Classes="monospaceFont selectableTextBlock Transparent" IsVisible="{Binding !ClipboardNotificationVisible}" Text="{Binding TransactionId, ConverterParameter=50, Converter={StaticResource LurkingWifeModeStringConverter}, Mode=OneWay}" Grid.Column="3" />
```

Comment:
I am OK with the solution when AutoCopy turned off - well done. 
However, when AutoCopy turned on I prefer the original behavior with the arrow cursor and the non-selectability.

Is it possible to make the `selectableTextBlock Transparent` in Classes conditional?